### PR TITLE
TRD-637 Tradelogs v2: Backfill Flow

### PR DESCRIPTION
**Pull request by linhnt3400**

### CODE REVIEW

File: v2/cmd/backfill/main.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	}
+
+	// trade log manager
+	storages := []storageTypes.Storage{
+		zxotcStorage.New(l, db),
+	}
+	manager := tradelogs.NewManager(l, storages)
+
+	// price filler
+	binanceClient := binance.NewClient(c.String(libapp.BinanceAPIKeyFlag.Name), c.String(libapp.BinanceSecretKeyFlag.Name))
```
Comment (ngocthanh1389) [inline v2/cmd/backfill/main.go:67]: calculate price in another service

File: v2/internal/worker/backfiller.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+		s:       s,
+		l:       l,
+		rpc:     rpc,
+	}
+}
+
+func (w *BackFiller) Run() error {
+	ticker := time.NewTicker(time.Hour)
+	defer ticker.Stop()
+	for ; ; <-ticker.C {
```
Comment (ngocthanh1389) [inline v2/internal/worker/backfiller.go:33]: run 1 time, add API to backfill from, to + exchange name

**Review by ngocthanh1389 — COMMENTED** [review]
Remove code relate to price filler

File: v2/cmd/backfill/main.go (hunk 1 above)
Comment (linhnt3400) [reply v2/cmd/backfill/main.go]: udpated

File: v2/internal/worker/backfiller.go (hunk 2 above)
Comment (linhnt3400) [reply v2/internal/worker/backfiller.go]: updated

**Pull request merged**